pages/04.Reference/06.images/docs.en.md

```diff
@@ -65,6 +69,16 @@ data "openstack_images_image_v2" "ubuntu-bionic" {
 }
 ```
 
+### Known issues with community images
```

Comment:
I wouldn't call it "community" because glance has a visibility status "community" and it might be confused with that

Reply:
Valid point